src/Main.c

```diff
@@ -1,10 +1,52 @@
 #include <stdio.h>
 #include <stdlib.h>
+#include<windows.h>
+#include<mysql.h>
 #include<string.h>
+MYSQL *oo,*conn;
+MYSQL_RES *read=NULL;
+MYSQL_RES *res=NULL;
+MYSQL_ROW row=NULL;
+
 
 char* login(int id, char pwd[25])
 {
-	return "admin";
+	char stmt[1500];
+	char qry[]={"select * from login_details where emp_id='%d'and password='%s'"};
+	oo=mysql_init(NULL);
+	mysql_real_connect(oo, "localhost", "root", "1234","payroll", 3306, NULL, 0);
+	if(oo)
+    {
+    	
+        int n=sprintf(stmt,qry,id,pwd);
+        mysql_query(oo,stmt);
+        read=mysql_store_result(oo);
+        row = mysql_fetch_row(read);
+        if(row==NULL)
+        {
+        	return "\nWrong username or password\n\n\n\n\n";
+        	
+		}
+		else
```

Comment:
Coding convention can be followed as: 

if (condition) { 
    // 4 space indent. 
} else if (...) { // The else goes on the same line as the closing brace.
   ... 
} else {
   ...
}